src/drum_kit.cairo

```diff
@@ -0,0 +1,185 @@
+use cairo_wave::note::{Note, NoteType};
+use cairo_wave::utils::{
+    generate_square_wave, generate_sawtooth_wave, generate_triangle_wave, generate_sine_wave
+};
+
+#[derive(Drop, Copy, Serde)]
+enum DrumSound {
+    Kick,
+    Snare,
+    Bass,
+    HiHat,
+}
+
+// Get drum sound
+fn get_drum_sound(sound: DrumSound, sample_rate: u32, bit_depth: u16) -> Array<u32> {
+    let result = match sound {
+        DrumSound::Kick => generate_kick(sample_rate, bit_depth),
+        DrumSound::Snare => generate_snare(sample_rate, bit_depth),
+        DrumSound::Bass => generate_bass(sample_rate, bit_depth),
+        DrumSound::HiHat => generate_hi_hat(sample_rate, bit_depth),
+    };
+
+    if result.is_empty() {
+        return array![1];
+    }
+    let mut non_zero_samples: Array<u32> = array![];
+    let mut i: u32 = 0;
+    loop {
+        if i >= result.len() {
+            break;
+        }
+        let sample = *result.at(i);
+        if sample == 0 {
```

Comment:
Do you imply there is am issue with zero samples?

Reply:
I left that because it gave me division by 0 errors at the beginning, as a precaution I kept it. I will do some tests to see if it works without that part now.